examples/LightStepTestUI/Pods/opentracing/Pod/Classes/OTSpanContext.h

```diff
@@ -17,22 +17,12 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol OTSpanContext
 
 /**
- * Add a baggage item that propagates along with the distributed trace. See
- * http://opentracing.io/spec/#baggage .
+ * An iterator for OTSpanContext baggage.
  *
- * @param key the key for the Baggage item
- * @param value: the Baggage item value; an opaque string value
+ * If the callback returns false, iteration stops and forEachBaggageItem:
+ * returns early.
  */
-- (void)setBaggageItem:(NSString*)key value:(NSString*)value;
```

Comment:
FYI, the big change at the OT level was that the `setBaggageItem` method moved to OTSpan rather than OTSpanContext, thus leaving the latter to become immutable.